dissect/target/plugins/os/unix/bsd/darwin/ios/_os.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import annotations
+
+import plistlib
+from dataclasses import dataclass
+from typing import Iterator
+
+from dissect.target.filesystem import Filesystem, VirtualFilesystem
+from dissect.target.helpers.record import COMMON_UNIX_FIELDS, TargetRecordDescriptor
+from dissect.target.plugin import OperatingSystem, export
+from dissect.target.plugins.os.unix.bsd.darwin._os import (
+    DarwinPlugin,
+    detect_macho_arch,
+)
+from dissect.target.target import Target
+
+IOSUserRecord = TargetRecordDescriptor(
```

Comment:
Why are the user records for iOS and macOS defined in their OS plugin files, but the application records are defined in `record.py`?

I think record definitions should primarily be in the file they are used in (so not `record.py`). I think originally the user records were defined in that file because they were used in a bunch of places and very high level OS stuff. I'm not sure why the application records are in there now, can't those be moved out?

The user records can maybe still go in there, it's convenient for example for `plugins/general/users.py` (which needs a type update for the new user records too).